### ISSUE TITLE
[Error] Java.Lang.IllegalArgumentException: Can only use lower 16 bits for requestCode

### ISSUE THREAD

**Issue by JaeWangL** [CrossGeeks/GoogleClientPlugin#33]
**Describe the bug**
When I run method 'LoginAsync()', "Can only use lower 16 bits for requestCode" is occured

**Screenshots**
```
Java.Lang.IllegalArgumentException: Can only use lower 16 bits for requestCode
  at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualVoidMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00089] in <fdf05f528e174febb3e55b587dbab368>:0 
  at Java.Interop.JniPeerMembers+JniInstanceMethods.InvokeVirtualVoidMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters) [0x0005d] in <fdf05f528e174febb3e55b587dbab368>:0 
  at Android.App.Activity.StartActivityForResult (Android.Content.Intent intent, System.Int32 requestCode) [0x00044] in <d9e99b55e96444569d54148253f9c212>:0 
  at Plugin.GoogleClient.GoogleClientManager.LoginAsync () [0x0003c] in <b6fce69c42c543f4b06726b36e946653>:0 
```

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S10e
 - OS: Android
 - Version 8.1

**Extra Version**
- Visual Studio: 2019
 - Xamarin.Forms: 3.6
-  Plugin.GoogleClient: 1.1.0

**Extra**
**- Please just modify 'AuthActivityID' in GoogleClientManager for Android**

**Comment by maxyhi** [comment]
did you find a solution for this issue?

**Comment by Pujolsluis** [comment]
Hey Guys! Thanks for reporting this issue will be working on it and releasing a fix on the next release.

**Comment by Pujolsluis** [comment]
Hey Guys!

This issue has been solved on the latest release v1.1.1, will proceed to close this issue.

Thanks for the report :)
